handlers/validator.go

```diff
@@ -183,21 +183,13 @@ func Validator(w http.ResponseWriter, r *http.Request) {
 				}
 			}
 
-			sumValid := uint64(0)
 			// check if a valid deposit exists
 			for _, d := range deposits.Eth1Deposits {
-				if d.ValidSignature {
-					sumValid += d.Amount
-				} else {
+				if !d.ValidSignature {
 					validatorPageData.Status = "deposited_invalid"
```

Comment:
**Nitpick:** You should add a `break` here:
```suggestion
					validatorPageData.Status = "deposited_invalid"
					break
```

Reply:
Sure, changed.
I also formatted a bit so that the two for loops look consistent.